### PR TITLE
Read CSRF token from meta tag, send with requests

### PR DESCRIPTION
**Pull request by nathanstitt**
The simplest thing that could possibly work.  I'm also a fan of making a `Tutor.boot()` method that could initialize this.

screen shot 2015-05-20 at 3 24 40 pm

### CODE REVIEW

**Comment by nathanstitt** [comment]
Corresponds with https://github.com/openstax/tutor-server/pull/300

**Comment by philschatz** [comment]
And needs an entry in #285

File: src/api.coffee [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 $ = require 'jquery'
 _ = require 'underscore'
-
+CSRF_Token = document.head.querySelector('meta[name=csrf-token]')?.getAttribute("content")
```
Comment (philschatz) [inline src/api.coffee]: Should an exception be thrown if this element is not found?

Comment (nathanstitt) [inline src/api.coffee]: Maybe, but if we do that then we'll also have to modify the dev server to set a dummy meta tag.  My thoughts were that the validation of the tag's presence and accuracy would be performed by the server when requests are performed.

I'd suggest that we split the difference and `console.warn` a "CRSF Token not detected" message.

**Pull request merged**